best_practices/business-logic.rst

```diff
@@ -122,6 +122,15 @@ library or strategy you want for this.
 
 In practice, many Symfony applications rely on the independent
 `Doctrine project`_ to define their model using entities and repositories.
+
+Doctrine support is not enabled by default in Symfony. So to use Doctrine
+as shown in the examples below you will need to install `Doctrine ORM support`_
+by executing the following command:
+
+.. code-block:: terminal
+
+    $ composer require "symfony/orm-pack"
```

Comment:
```suggestion
    $ composer require symfony/orm-pack
```

Reply:
The quotes on the composer line was to keep it consistent with the Data Fixtures section later on the page. In my opinion neither should have quotes but both having quotes is better than one with quotes and one without.

Reply:
Ah yes, I see. You could add a commit to remove the quotes over there too to make it consistent across the whole docs in my opinion.